Marlin/ultralcd.cpp

```diff
@@ -2799,16 +2804,17 @@ void lcd_update() {
 
         #if ENABLED(DOGLCD)  // Changes due to different driver architecture of the DOGM display
           static int8_t dot_color = 0;
-          dot_color = 1 - dot_color;
-          u8g.firstPage();
-          do {
+          if (!drawing_screen) {
+            u8g.firstPage();
+            dot_color = 1 - dot_color;
+          }
             lcd_setFont(FONT_MENU);
             u8g.setPrintPos(125, 0);
             u8g.setColorIndex(dot_color); // Set color for the alive dot
             u8g.drawPixel(127, 63); // draw alive dot
             u8g.setColorIndex(1); // black on white
```

Comment:
I realize that - but then it could be two dots? on indicating minimum buffer lever and the other maximum buffer level - so it would be an interval..